splunk_eventgen/eventgen_api_server/eventgen_server_api.py

```diff
@@ -371,7 +371,7 @@ def set_volume(self, target_volume):
                 stanza_num -= 1
             divided_volume = float(target_volume) / stanza_num
             for stanza, kv_pair in conf_dict.iteritems():
-                if isinstance(kv_pair, dict) and stanza != '.*' not in stanza:
+                if isinstance(kv_pair, dict) and 'global' not in stanza and '.*' not in stanza:
```

Comment:
maybe we should use `stanza != global and '.*' not in stanza` to avoid sample name with `global` in it for example "[test_global_sample]".